echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/slack/SlackNotificationService.groovy

```diff
@@ -51,12 +51,15 @@ class SlackNotificationService implements NotificationService {
   EchoResponse.Void handle(Notification notification) {
     def text = notificationTemplateEngine.build(notification, NotificationTemplateEngine.Type.BODY)
     notification.to.each {
+      def response
```

Comment:
FWIW, there was a previous request to change this to "Response response" but that doesn't compile ("Response" isn't a type, apparently).  So I put this back as it was.

Reply:
you probably need `import retrofit.client.Response` at the top. Then it should be fine with `Response response`

Reply:
but this is minor, happy to merge as is